app/controllers/api/v1/base_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Api
+  module V1
```

Comment:
__[Reek]__

[UncommunicativeModuleName] Api::V1 has the name 'V1'


<!-- Comment by [SideCI](https://sideci.com) -->